Add custom prism theme

hursey013 opened the pull request:
## Changes proposed in this pull request:
- Uses lighter theme for syntax highlighting 

Before:
<img width="637" alt="Screenshot 2025-04-11 at 12 18 56 PM" src="https://github.com/user-attachments/assets/68c659a0-9efd-444c-b582-a6083fd12042" />

After:
<img width="637" alt="Screenshot 2025-04-11 at 12 14 42 PM" src="https://github.com/user-attachments/assets/2d67a98a-9afb-49bb-a84a-8f6cb68e55dc" />

## security considerations
None

markdboyd commented:
I kind of prefer the existing theme. I think the contrast of the dark background and lighter text is a bit easier to read. And the different background color of the `<` and `>` is less readable, to me

kfoley-18F commented:
Would it be helpful to get an accessibility expert to review and make sure it'll meet Section 508 requirements? 
Holler if so.

hursey013 commented:
The different background color behind the `<` and `>` operators was my fault, fixed that. Light vs dark theme feels like it could be personal preference, so it may be worth exploring offering a light and dark theme for `/docs` as part of splitting them off into their own separate website. After running it by @sknep, proposing moving forward with this in the meantime

Updated:
![Screenshot 2025-04-15 at 3 02 50 PM](https://github.com/user-attachments/assets/c207d214-7a93-4014-80b0-bbd724bcc9f7)

markdboyd commented:
@hursey013 Looks good to me!

sknep reviewed (approved):
Dunno if i count in codeowners but i think i should?

markdboyd commented:
@sknep @hursey013 Do we need a "studio" team in GitHub which would be another codeowner of this repo?
